static/js/theme-map/CardListenerAssigner.js

```diff
@@ -9,6 +9,20 @@ class CardListenerAssigner {
      * @type {Answers.Component}
      */
     this.card = card;
+
+    /**
+     * The upper bound (inclusive) of the mobile breakpoint in px
+     *
+     * @type {Number}
+     */
+    this.mobileBreakpointMax = 991;
+
+    /**
+     * The matcher to determine if the window width is within the mobile breakpoint
+     *
+     * @type {MediaQueryList}
+     */
+    this.mediaMatcher = window.matchMedia(`(max-width: ${this.mobileBreakpointMax}px)`);
```

Comment:
could we name this mobileMediaMatcher or something that makes it obvious it this.mediaMatcher.matches returns true for mobile?

Reply:
Good point! Changed